coc-committee-charter-proposed.md

```diff
@@ -57,9 +57,9 @@ Any individual who serves as an alternate on an ongoing basis (rather than for a
 
 When a Code of Conduct incident is subject to both the CNCF Code of Conduct and the Linux Foundation Events Code of Conduct, the Linux Foundation may designate a liaison (the “LF Liaison”) to coordinate efforts between the Linux Foundation and the CoC Committee. The LF Liaison may participate in CoC Committee discussions and meetings regarding the incident on a non-voting basis. The LF Liaison shall comply with the [Confidentiality Policy](https://github.com/cncf/wg-coc/blob/main/coc-incident-resolution-procedures-proposed.md#confidentiality-policy).
 
-## Governing Board Liaison
+## CNCF Governing Board Liaison
```

Comment:
a couple of nits @joannalee333 !

- `shall not participate in the investigation` -> `shall not participate in any investigations`
- `regading` -> `regarding`

Reply:
Thanks, @dims. I've fixed those

Reply:
🙏